crates/proof/executor/src/executor/mod.rs

```diff
@@ -534,13 +534,9 @@ mod test {
     // }
 
     #[rstest]
-    #[case::small_block(22884230)]
-    #[case::small_block_2(22884231)]
-    #[case::small_block_3(22880574)]
-    #[case::small_block_4(22887258)]
-    #[case::medium_block(22886464)]
-    #[case::medium_block_2(22886311)]
-    #[case::medium_block_3(22880944)]
+    #[case::small_block(10311000)] // Unichain Mainnet
+    #[case::small_block_2(10211000)] // Unichain Mainnet
+    #[case::small_block_3(10215000)] // Unichain Mainnet
```

Comment:
@clabby what are your thoughts here?

Reply:
Do we have a guide yet on how to produce this data? Would be super helpful.

Reply:
Yeah we really should generate more cases here (and delete the old ones - I see all of these deleted ones still exist in `testdata`.) It's totally okay to cycle these out every now and then, though, since it's impossible to regenerate these old cases without a blob archiver.

There's a short guide in the commented-out test above, regenerating is pretty easy:
https://github.com/op-rs/kona/blob/263c8a03bf7ee1b9b6d37fce1250eac91027c009/crates/proof/executor/src/executor/mod.rs#L524-L534